tests/v2/validation/provisioning/hosted/aks/hosted_provisioning_test.go

```diff
@@ -134,19 +117,33 @@ func (h *HostedAKSClusterProvisioningTestSuite) testProvisioningHostedAKSCluster
 	require.NoError(h.T(), err)
 	assert.NotEmpty(h.T(), clusterToken)
 
+	err = nodestat.IsNodeReady(rancherClient, clusterResp.ID)
+	require.NoError(h.T(), err)
+
+	podResults, podErrors := pods.StatusPods(rancherClient, clusterResp.ID)
+	assert.NotEmpty(h.T(), podResults)
+	assert.Empty(h.T(), podErrors)
+
 	return clusterResp, nil
 }
 
-func (h *HostedAKSClusterProvisioningTestSuite) testScalingAKSNodePools(rancherClient *rancher.Client, clusterName string) {
+func (h *HostedAKSClusterProvisioningTestSuite) testScalingAKSNodePools(rancherClient *rancher.Client, clusterName string, cluster *management.Cluster) {
 	cloudCredential, err := azure.CreateAzureCloudCredentials(rancherClient)
```

Comment:
If we run the whole test suite it's bit redundant to create another cloud credential. I think we should do something similar like with cluster.

Reply:
Makes sense to me, I'll try to have it on parity with that.